editor/connections_dialog.cpp

```diff
@@ -891,6 +898,12 @@ ConnectDialog::ConnectDialog() {
 	one_shot->set_tooltip_text(TTR("Disconnects the signal after its first emission."));
 	hbox->add_child(one_shot);
 
+	append_source = memnew(CheckBox);
+	append_source->set_h_size_flags(0);
+	append_source->set_text(TTR("Append source"));
+	append_source->set_tooltip_text(TTR("The source object is automatically sent when the signal is emitted."));
+	vbc_right->add_child(append_source);
```

Comment:
You could change `hbox` into a FlowContainer instead.

Reply:
Changed to FlowContainer.